packages/docs/docs/concepts/cornerstone-core/viewports.md

```diff
@@ -20,6 +20,34 @@ viewed via 4 viewports in a “4-up” view: Axial MPR, Sagittal MPR, Coronal MP
 
 </div>
 
+## Initial State
+
+The initial state of the viewport can be controlled by setting the default options
+initialRange, initialCenter and canvasCenter, all point values.
+
+### initialRange
+
+The initial range is the fraction of the image size in width and height that should
+be scaled to fit the viewport. For example, the range [0.5,2] means to fit
+half the width into the viewport and twice the height, whichever one causes the
+total scale to be smaller.
+
+### initialCenter and canvasCenter
+
+The initialCenter and canvasCenter are percentage of image size and canvas size,
+such that the two points correspond initially. That is, suppose the
+initial center is [0.5, 1] and the canvas center is 0.5,0.5, with a 512x512 image
+on a 1024x800 canvas, then the image point 256,512 should be displayed overtop
+the center of the canvas (at 512,400). This allows full control over alignment
+of images such as mammography images, without needing to know the actual size
+of the canvas and image beforehand.
+
+## Initial State helper functions
+
+There area few functions that act on the viewport relative to the initial state.
+The zoom function change the size of hte image relative to the initial zoom of 1,
+while the pan functions change the pan relative to the initial pan of [0,0].
+
```

Comment:
This document is a top level explanation of the viewports. We don't need to add complexity by explaining these optional arguments (similar to how we don't explain background, orientation etc. here). These comments should be moved to the Viewport (class) on top of the property for it to be properly shown in the api doc (see here https://www.cornerstonejs.org/api/core/class/StackViewport).